### PR TITLE
Mark as packages non-stable for 1.1.5

### PR DESCRIPTION
**Pull request by safern**
cc: @weshaggard

### CODE REVIEW

File: src/Microsoft.DotNet.PlatformAbstractions/project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for making code that uses file system and environment testable.",
-  "version": "1.1.4-servicing-*",
```
Comment (weshaggard) [inline src/Microsoft.DotNet.PlatformAbstractions/project.json]: We haven't been shipping PlatformAbstractions and DependencyModel so lets not bump there versions until we think we need to ship them (we may even want to consider downgrading them to one higher then the last one we shipped).

cc @eerhardt

Comment (safern) [reply src/Microsoft.DotNet.PlatformAbstractions/project.json]: Ok. Will fix that then. So 1.1.4-servicing-* is good? Or should we downgrade to 1.1.3?

Comment (weshaggard) [reply src/Microsoft.DotNet.PlatformAbstractions/project.json]: Just revert to 1.1.4 for now.

Comment (safern) [reply src/Microsoft.DotNet.PlatformAbstractions/project.json]: Ok

Comment (eerhardt) [reply src/Microsoft.DotNet.PlatformAbstractions/project.json]: This sounds like a good plan to me.

Note, for release/2.0.0, when we merge in [the build changes from master](https://github.com/dotnet/core-setup/pull/3235), these packages will automatically have the version number that the whole repo uses.  We won't have these versions spread all over the repo.

Of course, we will be able to override the version number for each package, but by default these numbers won't be duplicated.

**Comment by safern** [comment]
test Ubuntu x64 Release Build

**Comment by safern** [comment]
Oops. Closed it unintentionally

**Pull request merged**